sw/device/silicon_creator/manuf/lib/individualize_sw_cfg.c

```diff
@@ -91,6 +91,32 @@ static status_t otp_img_write(const dif_otp_ctrl_t *otp,
   return OK_STATUS();
 }
 
+static status_t otp_img_read(dif_otp_ctrl_partition_t partition,
```

Comment:
shouldn't this function just read out a specific field? why do we need to read out all fields. Don't we just want to read out the fields that are not provisioned until the end so we know what their expected values are?

Also, could we rename this `otp_expected_value_read`? or something more descript? currently, the name seems to indicate it is reading from OTP directly, not from what is expected to be written to OTP, i.e, what the values baked into the SW image itself.

Reply:
I've updated the commit to explicitly update the desired fields, let me know if it looks good to you. 

Renamed the function to `otp_img_expected_value_read`